src/oaklib/implementations/sqldb/sql_implementation.py

```diff
@@ -671,6 +672,29 @@ def entity_metadata_map(self, curie: CURIE, include_all_triples=False) -> METADA
         self.add_missing_property_values(curie, m)
         return dict(m)
 
+    def entities_metadata_statements(
+        self, curies: Iterable[CURIE], predicates: Optional[List[PRED_CURIE]] = None
+    ) -> Iterator[METADATA_STATEMENT]:
+        q = self.session.query(Statements)
+        # if not include_all_triples:
+        #     subquery = self.session.query(RdfTypeStatement.subject).filter(
+        #         RdfTypeStatement.object == "owl:AnnotationProperty"
+        #     )
+        #     annotation_properties = {row.subject for row in subquery}
+        #     annotation_properties = annotation_properties.union(STANDARD_ANNOTATION_PROPERTIES)
+        #     q = q.filter(Statements.predicate.in_(tuple(annotation_properties)))
+        q = q.filter(Statements.subject.in_(curies))
+        if predicates is not None:
+            q = q.filter(Statements.predicate.in_(predicates))
+        for row in q:
+            if row.value is not None:
+                v = _python_value(row.value, row.datatype)
+            elif row.object is not None:
+                v = row.object
+            else:
+                v = None
+            yield row.subject, row.predicate, v, row.datatype, {}
```

Comment:
remember to include axiom annotations - consider making this optional, to allow for faster processing of the basic metadata

Reply:
DOne. You mean just uncommenting the lines above and adding `include_all_triples` as a param, coorect?